cli/CHANGELOG.md

```diff
@@ -34,6 +34,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 
 - `transaction` and `transactions` indexed transactions in opposite order;
+- Enter doesn't show helper anymore;
+- Earlier trim of the input so that pre-commands (`help`, `clear`, `accounts`, ...) work even with leading/trailing spaces;
```

Comment:
I am not sure this (line 38) is a bug tbh and should be under the "Fixed" tab. To me it's more like a changed behavior?

Reply:
I am not against trimming btw ... I am saying that the behavior will for most people diverge from what they are used to in their terminal...and those terminals probably don't trim leading and trailing whitespace for a (good?) reason.

Reply:
You can definitely do `         cd..` in fish, just tested it.

Reply:
Idk, this really seems like a bug to me, there is absolutely no reason not to trim?

Reply:
you're right ... when I tested I copied from here which didn't include the space in between :see_no_evil: